packages/lit-html/src/static.ts

```diff
@@ -9,6 +9,43 @@
 
 import {html as coreHtml, svg as coreSvg, TemplateResult} from './lit-html.js';
 
+interface StaticValue {
+  /** The value to interpolate as-is into the template. */
+  _$litStatic$: string;
+
+  /**
+   * A value that can't be decoded from ordinary JSON, make it harder for
+   * a attacker-controlled data that goes through JSON.parse to produce a valid
+   * StaticValue.
+   */
+  r: typeof brand;
+}
+
+/**
+ * Prevents JSON injection attacks.
+ *
+ * The goals of this brand:
+ *   1) fast to check
+ *   2) code is small on the wire
+ *   3) multiple versions of Lit in a single page will all produce mutually
```

Comment:
How does this work? Is it possible to add this case as a test?

Reply:
It's difficult to add tests for multiple versions. It works before `Symbol.for(s)` uses a global cache keyed on `s`, so `Symbol.for('x') === Symbol.for('x')`

Reply:
Oh! TIL!